testdata/default/cheats/RpcUrls.t.sol

```diff
@@ -9,8 +9,8 @@ contract RpcUrlTest is DSTest {
 
     // returns the correct url
     function testCanGetRpcUrl() public {
-        string memory url = vm.rpcUrl("rpcAlias"); // note: this alias is pre-configured in the test runner
-        assertEq(url, "https://eth-mainnet.alchemyapi.io/v2/Lc7oIGYeL_QvInzI0Wiu_pOZZDEKBrdf");
+        string memory url = vm.rpcUrl("mainnet");
+        assertEq(bytes(url).length, 69);
```

Comment:
I typed 69 as a placeholder so i can see what the actual value is, but it's actually right